graylog2-web-interface/src/views/hooks/BindSearchParamsFromQuery.test.ts

```diff
@@ -63,6 +63,12 @@ describe('BindSearchParamsFromQuery should', () => {
     expect(QueriesActions.update).not.toHaveBeenCalled();
   });
 
+  it('not update query when query is already up to date', async () => {
+    await bindSearchParamsFromQuery(defaultInput);
+
+    expect(QueriesActions.update).not.toHaveBeenCalled();
+  });
+
```

Comment:
This test is not testing the described bug, but it still provides a benefit. I would add a filter specific test here, if the `toBuilder` method of the `Query` would still contain filter specific logic.